### PR TITLE
doc: Initial release notes for v1.9

### PR DESCRIPTION
**Pull request by Shilpashree3lc-zz**
doc: Initial release notes for v1.9

Initial draft of the release notes for Zephyr v1.9 release which contains some interesting changes on Kernel, Architecture, Boards, Drivers and Sensors, Networking, Bluetooth, documentation, and jira items. etc..

Signed-off-by: Shilpashree L C shilpashree.lc@intel.com

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]
Some trailing white space and misspellings caught (as noted here: https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/2258/consolidateTests). 

Also, I'm not sure it's that useful to start the release notes this early in the process.  Since most of this material comes from either GitHub or Jira queries, it can be done closer to the release to capture current information (and to fix spelling errors once).

File: doc/release-notes-1.9.rst [hunk 1]
```diff
@@ -33,61 +33,512 @@ The following sections provide detailed lists of changes by component.
 Kernel
 ******
 
-* change description
+* kernel: add early init routines for app RAM
+* kernel: add config for app/kernel split
+* kernel: queue
+* kernel: remove gdb_server
+* kernel: make K_.*_INITIALIZER private to kernel
```
Comment (dbkinder) [inline doc/release-notes-1.9.rst]: the text `K_` looks like a link reference to Sphinx (and it gives an error saying k is an unknown reference). Need to instead write this as `K\_`

**Comment by pfalcon** [comment]
For me, this doesn't look like "containing some interesting changes", but rather just unfiltered commit subject list. To be similar to previous release notes, this would need to be condensed 3-4 times.

**Comment by dbkinder** [comment]
Some spelling errors in commit/PR titles are causing problems

**Review by nashif — CHANGES_REQUESTED** [review]
agree with @pfalcon , this is just a commit list, we do not need this level of details, instead we need to get the highlights and summary of changes.

**Comment by nashif** [comment]
outdated

**Pull request closed without merging**